js/index.js

```diff
@@ -1,6 +1,8 @@
-'use strict'
+'use strict';
 
-import { NativeModules } from 'react-native'
+import { NativeModules } from 'react-native';
+import { Rokt, RoktConfigBuilder, CacheConfig } from './rokt/rokt';
```

Comment:
I think you can remove this now since we are no longer using the builder.  you'll have to import 
```suggestion
import { Rokt, RoktConfig, CacheConfig } from './rokt/rokt';
```